### PR TITLE
Increase SbMediaConfigurationTest perf requirement

### PR DESCRIPTION
**Pull request by jasonzhangxx**
b/284140486

### CODE REVIEW

File: starboard/android/shared/media_get_audio_configuration.cc [hunk 1]
```diff
@@ -40,13 +40,6 @@ bool SbMediaGetAudioConfiguration(
     return false;
   }
 
-  bool result = MediaCapabilitiesCache::GetInstance()->GetAudioConfiguration(
+  return MediaCapabilitiesCache::GetInstance()->GetAudioConfiguration(
       output_index, out_configuration);
-
-  SB_LOG(INFO) << "Audio connector type for index " << output_index << " is "
```
Comment (xiaomings) [inline starboard/android/shared/media_get_audio_configuration.cc:46]: I'd assume that the reason to remove the log is to reduce the log spamming in the benchmark, as the function will be run many times in the benchmark.
The log is useful for debugging purposes, we should keep it as is.  We can further reduce the number of calls in the benchmark, or refine the benchmark infra.

Comment (jasonzhangxx) [reply starboard/android/shared/media_get_audio_configuration.cc:46]: The log significantly impacts performance. Removing the log can reduce the average executing time of SbMediaGetAudioConfiguration() by ~1ms on Sabrina.

Comment (jasonzhangxx) [reply starboard/android/shared/media_get_audio_configuration.cc:46]: There's another pr https://github.com/youtube/cobalt/pull/1784 to temporarily change the log level for performance tests. With that change, we may not need to remove the log.

Comment (jasonzhangxx) [reply starboard/android/shared/media_get_audio_configuration.cc]: With #1784 merged, we no longer need to remove the log.

**Review by xiaomings — APPROVED** [review]

**Pull request merged**